packages/grid/x-data-grid-premium/src/hooks/features/aggregation/gridAggregationFunctions.ts

```diff
@@ -2,12 +2,12 @@ import { GridValueFormatterParams } from '@mui/x-data-grid-pro';
 import { isNumber } from '@mui/x-data-grid-pro/internals';
 import { GridAggregationFunction } from './gridAggregationInterfaces';
 
-const sumAgg: GridAggregationFunction<number> = {
+const sumAgg: GridAggregationFunction<unknown, number> = {
```

Comment:
We don't control the values passed here, so typing them as numbers here is too naive.

Reply:
This applies to other aggregation functions, I'll address them in separate PRs

Reply:
Will this make the change `breaking` by any chance? 🤔

Reply:
I don't think so.
Even if someone is calling the `sumAgg` function directly, the type is now wider.
Do you see other potential issues?

Reply:
Nope, all good 👍